### ISSUE TITLE
Can't add slack integration

### ISSUE THREAD

**Issue by AndreyLevchenko** [aquasecurity/postee#113]
I'm trying to add Slack integration and I see this:
![slack-integration-issue](https://user-images.githubusercontent.com/353652/114162677-2490d300-994b-11eb-9355-95a0fdcdf6b5.png)
but I can't find "Incoming Webhooks" or "Add to Slack" mentioned in Readme. Seems Slack dropped some support for Custom Integrations and new plugin needs to be added.

**Comment by AndreyLevchenko** [comment]
I did more investigations and figured out that no code changes are required, but configuration steps are different:
I used steps below:

1. Visit api.slack.com
2. Press "Create custom app"
3. Fill app name and select slack workspace
4. Open "Incoming webhooks" tab
5. Enable "Incoming webhooks"
6. Add webhook to workspace
7. On next screen pick slack channel and click allow
8. Copy webhook url to the Postee config

After these steps integration is working properly. So minimal changes are just modifying `Readme.md`with steps above. Also official Postee app can be created/approved on Slack marketplace
